Tasks/SshV0/task.json

```diff
@@ -143,6 +143,14 @@
             "argumentFormat": ""
         }
     },
+    "restrictions": {
```

Comment:
Since this task could execute custom script on remote machine and output vso commands - do we really need to restrict it?